### PR TITLE
Second PR For Readme Updates from AAD To Microsoft Entra ID

### PR DESCRIPTION
**Pull request by Mohammed-MSFT**

### CODE REVIEW

File: samples/app-sso/csharp/README.md [hunk 1]
```diff
@@ -33,7 +33,7 @@ This bot has been created using [Bot Framework](https://dev.botframework.com), i
...
 
-The sample uses the bot authentication capabilities in [Azure Bot Service](https://docs.botframework.com), providing features to make it easier to develop a bot that authenticates users to various identity providers such as Azure AD (Azure Active Directory), GitHub, Uber, etc. The OAuth token is then used to make basic Microsoft Graph queries. Refer the **SSO** setup [documentation](https://docs.microsoft.com/microsoftteams/platform/bots/how-to/authentication/add-authentication?tabs=dotnet%2Cdotnet-sample).
+The sample uses the bot authentication capabilities in [Azure Bot Service](https://docs.botframework.com), providing features to make it easier to develop a bot that authenticates users to various identity providers such as (Microsoft Entra ID), GitHub, Uber, etc. The OAuth token is then used to make basic Microsoft Graph queries. Refer the **SSO** setup [documentation](https://docs.microsoft.com/microsoftteams/platform/bots/how-to/authentication/add-authentication?tabs=dotnet%2Cdotnet-sample).
```
Comment (Pawank-MSFT) [inline samples/app-sso/csharp/README.md:36]: (Microsoft Entra ID)  >> Microsoft Entra ID

File: samples/app-sso/nodejs/README.md [hunk 2]
```diff
@@ -30,7 +30,7 @@ This bot has been created using [Bot Framework](https://dev.botframework.com), i
...
 
-The sample uses the bot authentication capabilities in [Azure Bot Service](https://docs.botframework.com), providing features to make it easier to develop a bot that authenticates users to various identity providers such as Azure AD (Azure Active Directory), GitHub, Uber, etc. The OAuth token is then used to make basic Microsoft Graph queries. Refer the **SSO** setup [documentation](https://docs.microsoft.com/microsoftteams/platform/bots/how-to/authentication/add-authentication?tabs=node-js%2Cnode-js-dialog-sample).
+The sample uses the bot authentication capabilities in [Azure Bot Service](https://docs.botframework.com), providing features to make it easier to develop a bot that authenticates users to various identity providers such as Azure AD (Microsoft Entra ID), GitHub, Uber, etc. The OAuth token is then used to make basic Microsoft Graph queries. Refer the **SSO** setup [documentation](https://docs.microsoft.com/microsoftteams/platform/bots/how-to/authentication/add-authentication?tabs=node-js%2Cnode-js-dialog-sample).
```
Comment (Pawank-MSFT) [inline samples/app-sso/nodejs/README.md:33]: Azure AD (Microsoft Entra ID) >> Microsoft Entra ID

File: samples/bot-conversation-sso-quickstart/csharp_dotnetcore/BotConversationSsoQuickstart/README.md [hunk 3]
```diff
@@ -21,7 +21,7 @@ This bot has been created using [Bot Framework](https://dev.botframework.com), i
...
 
-The sample uses the bot authentication capabilities in [Azure Bot Service](https://docs.botframework.com), providing features to make it easier to develop a bot that authenticates users to various identity providers such as Azure AD (Azure Active Directory), GitHub, Uber, etc. The OAuth token is then used to make basic Microsoft Graph queries.
+The sample uses the bot authentication capabilities in [Azure Bot Service](https://docs.botframework.com), providing features to make it easier to develop a bot that authenticates users to various identity providers such as Azure AD (Microsoft Entra ID), GitHub, Uber, etc. The OAuth token is then used to make basic Microsoft Graph queries.
```
Comment (Pawank-MSFT) [inline samples/bot-conversation-sso-quickstart/csharp_dotnetcore/BotConversationSsoQuickstart/README.md:24]: Azure AD (Microsoft Entra ID) >> Microsoft Entra ID

File: samples/bot-conversation-sso-quickstart/js/README.md [hunk 4]
```diff
@@ -21,7 +21,7 @@ This bot has been created using [Bot Framework](https://dev.botframework.com), i
...
 
-The sample uses the bot authentication capabilities in [Azure Bot Service](https://docs.botframework.com), providing features to make it easier to develop a bot that authenticates users to various identity providers such as Azure AD (Azure Active Directory), GitHub, Uber, etc. The OAuth token is then used to make basic Microsoft Graph queries.
+The sample uses the bot authentication capabilities in [Azure Bot Service](https://docs.botframework.com), providing features to make it easier to develop a bot that authenticates users to various identity providers such as Azure AD (Microsoft Entra ID), GitHub, Uber, etc. The OAuth token is then used to make basic Microsoft Graph queries.
```
Comment (Pawank-MSFT) [inline samples/bot-conversation-sso-quickstart/js/README.md:24]: Azure AD (Microsoft Entra ID) >> Microsoft Entra ID

File: samples/bot-daily-task-reminder/nodejs/Readme.md [hunk 5]
```diff
@@ -62,7 +62,7 @@ The simplest way to run this sample in Teams is to use Teams Toolkit for Visual
 > the Teams service needs to call into the bot.
 
 1) Setup for Bot
-- In Azure portal, create AAD app registraion and it will generate MicrosoftAppId and MicrosoftAppPassword for you.
+- In Azure portal, create Microsoft Entra app registraion and it will generate MicrosoftAppId and MicrosoftAppPassword for you.
```
Comment (Pawank-MSFT) [inline samples/bot-daily-task-reminder/nodejs/Readme.md:65]: Microsoft Entra ID

File: samples/bot-daily-task-reminder/nodejs/Readme.md [hunk 6]
```diff
@@ -100,7 +100,7 @@ The simplest way to run this sample in Teams is to use Teams Toolkit for Visual
     npm install
     ```
 
-- Update the `.env` configuration file in your project folder for the bot to use the `MicrosoftAppId`, `MicrosoftAppPassword` (Note the MicrosoftAppId is the AppId created while doing AAD app registration in Azure portal, the MicrosoftAppPassword is referred to as the "client secret" generated while creating Secret in AAD app registration.
+- Update the `.env` configuration file in your project folder for the bot to use the `MicrosoftAppId`, `MicrosoftAppPassword` (Note the MicrosoftAppId is the AppId created while doing Microsoft Entra app registration in Azure portal, the MicrosoftAppPassword is referred to as the "client secret" generated while creating Secret in Microsoft Entra ID app registration.
```
Comment (Pawank-MSFT) [inline samples/bot-daily-task-reminder/nodejs/Readme.md:103]: doing Microsoft Entra >> doing Microsoft Entra ID

File: samples/bot-formatting-cards/csharp/README.md [hunk 7]
```diff
@@ -132,7 +132,7 @@ Please find below demo manifest which is deployed on Microsoft Azure and you can
         ```
    
    **Note: In adaptive card, what we are defining (User details) should be exist in the same tenant where you are testing the app (teams' login) etc...**
-  - Update the user AAD object ID in your adaptive card JSON from your tenant's AAD users available in the Azure portal.
+  - Update the user Microsoft Entra object ID in your adaptive card JSON from your tenant's Microsoft Entra users available in the Azure portal.
```
Comment (Pawank-MSFT) [inline samples/bot-formatting-cards/csharp/README.md:135]: Microsoft Entra users >> Microsoft Entra ID users

File: samples/bot-formatting-cards/nodejs/Readme.md [hunk 8]
```diff
@@ -120,7 +120,7 @@ The simplest way to run this sample in Teams is to use Teams Toolkit for Visual
         ```
 
   **Note: In adaptive card, what we are defining (User details) should be exist in the same tenant where you are testing the app (teams' login) etc...**
-  - Update the user AAD object ID in your adaptive card JSON from your tenant's AAD users available in the Azure portal.
+  - Update the user Microsoft Entra object ID in your adaptive card JSON from your tenant's Microsoft Entra users available in the Azure portal.
```
Comment (Pawank-MSFT) [inline samples/bot-formatting-cards/nodejs/Readme.md:123]: Microsoft Entra users >> Microsoft Entra ID users

File: samples/bot-teams-authentication/csharp/README.md [hunk 9]
```diff
@@ -20,13 +20,13 @@ This bot has been created using [Bot Framework](https://dev.botframework.com), i
...
 
-The sample uses the bot authentication capabilities in [Azure Bot Service](https://docs.botframework.com), providing features to make it easier to develop a bot that authenticates users to various identity providers such as Azure AD (Azure Active Directory), GitHub, Uber, etc. The OAuth token is then used to make basic Microsoft Graph queries.
+The sample uses the bot authentication capabilities in [Azure Bot Service](https://docs.botframework.com), providing features to make it easier to develop a bot that authenticates users to various identity providers such as Azure AD (Microsoft Entra ID), GitHub, Uber, etc. The OAuth token is then used to make basic Microsoft Graph queries.
```
Comment (Pawank-MSFT) [inline samples/bot-teams-authentication/csharp/README.md:23]: Azure AD (Microsoft Entra ID) >> Microsoft Entra ID

File: samples/bot-teams-authentication/java/README.md [hunk 10]
```diff
@@ -21,7 +21,7 @@ This bot has been created using [Bot Framework](https://dev.botframework.com), i
...
 
-The sample uses the bot authentication capabilities in [Azure Bot Service](https://docs.botframework.com), providing features to make it easier to develop a bot that authenticates users to various identity providers such as Azure AD (Azure Active Directory), GitHub, Uber, etc. The OAuth token is then used to make basic Microsoft Graph queries.
+The sample uses the bot authentication capabilities in [Azure Bot Service](https://docs.botframework.com), providing features to make it easier to develop a bot that authenticates users to various identity providers such as Azure AD (Microsoft Entra ID), GitHub, Uber, etc. The OAuth token is then used to make basic Microsoft Graph queries.
```
Comment (Pawank-MSFT) [inline samples/bot-teams-authentication/java/README.md:24]: Azure AD (Microsoft Entra ID) > Microsoft Entra ID

File: samples/bot-teams-authentication/nodejs/README.md [hunk 11]
```diff
@@ -20,7 +20,7 @@ This bot has been created using [Bot Framework](https://dev.botframework.com), i
...
 
-The sample uses the bot authentication capabilities in [Azure Bot Service](https://docs.botframework.com), providing features to make it easier to develop a bot that authenticates users to various identity providers such as Azure AD (Azure Active Directory), GitHub, Uber, etc. The OAuth token is then used to make basic Microsoft Graph queries.
+The sample uses the bot authentication capabilities in [Azure Bot Service](https://docs.botframework.com), providing features to make it easier to develop a bot that authenticates users to various identity providers such as Azure AD (Microsoft Entra ID), GitHub, Uber, etc. The OAuth token is then used to make basic Microsoft Graph queries.
```
Comment (Pawank-MSFT) [inline samples/bot-teams-authentication/nodejs/README.md:23]: Azure AD (Microsoft Entra ID) > Microsoft Entra ID

File: samples/bot-teams-authentication/python/README.md [hunk 12]
```diff
@@ -22,7 +22,7 @@ This bot has been created using [Bot Framework](https://dev.botframework.com), i
...
 
-The sample uses the bot authentication capabilities in [Azure Bot Service](https://docs.botframework.com), providing features to make it easier to develop a bot that authenticates users to various identity providers such as Azure AD (Azure Active Directory), GitHub, Uber, etc. The OAuth token is then used to make basic Microsoft Graph queries.
+The sample uses the bot authentication capabilities in [Azure Bot Service](https://docs.botframework.com), providing features to make it easier to develop a bot that authenticates users to various identity providers such as Azure AD (Microsoft Entra ID), GitHub, Uber, etc. The OAuth token is then used to make basic Microsoft Graph queries.
```
Comment (Pawank-MSFT) [inline samples/bot-teams-authentication/python/README.md:25]: Azure AD (Microsoft Entra ID) > Microsoft Entra ID

File: samples/graph-appcatalog-lifecycle/nodejs/README.md [hunk 13]
```diff
@@ -170,7 +170,7 @@ Click on Add Permissions to commit your changes.
 
     - Enter a name for your new Connection setting. This will be the name that gets referenced inside the settings of your bot service code in step 5
 
-    - In the Service Provider dropdown, select Azure Active Directory V2
+    - In the Service Provider dropdown, Azure Active Directory V2
```
Comment (Pawank-MSFT) [inline samples/graph-appcatalog-lifecycle/nodejs/README.md:173]: missed "select" here

File: samples/graph-rsc/csharp/README.md [hunk 14]
```diff
@@ -43,7 +43,7 @@ Please find below demo manifest which is deployed on Microsoft Azure and you can
 
 ## Setup
 
-1) Register your app with Microsoft identity platform via the Azure AD portal (AAD app registration)
+1) Register your app with Microsoft identity platform via the Azure AD portal (Microsoft Entra ID app registration)
```
Comment (Pawank-MSFT) [inline samples/graph-rsc/csharp/README.md:46]: Azure AD portal (Microsoft Entra ID app registration) >> Microsoft Entra ID portal (Microsoft Entra ID app registration)

File: samples/graph-rsc/csharp/README.md [hunk 15]
```diff
@@ -72,14 +72,14 @@ Please find below demo manifest which is deployed on Microsoft Azure and you can
    ```
 
 5)  Update appsettings.json
-    - Update configuration for `<<Client Id>>`, `<<Client Secret>>` and ``<<Tenant Id>>`` with the ```MicrosoftAppId``` ,  ```MicrosoftAppPassword``` and ```TenantId``` which was generated while doing AAD pp registration in your Azure Portal.
+    - Update configuration for `<<Client Id>>`, `<<Client Secret>>` and ``<<Tenant Id>>`` with the ```MicrosoftAppId``` ,  ```MicrosoftAppPassword``` and ```TenantId``` which was generated while doing Microsoft Entra ID pp registration in your Azure Portal.
```
Comment (Pawank-MSFT) [inline samples/graph-rsc/csharp/README.md:75]: Microsoft Entra ID pp registration >> Microsoft Entra ID app registration

File: samples/msgext-search-auth-config/csharp/README.md [hunk 16]
```diff
@@ -85,7 +85,7 @@ Refer to [Bot SSO Setup document](https://github.com/OfficeDev/Microsoft-Teams-S
 
 1) Update the `appsettings.json` configuration for the bot to use the MicrosoftAppId, MicrosoftAppPassword, MicrosoftAppTenantId generated in Step 1 (App Registration creation). (Note the App Password is referred to as the "client secret" in the azure portal and you can always create a new client secret anytime.)
     - Set "MicrosoftAppType" in the `appsettings.json`. (**Allowed values are: MultiTenant(default), SingleTenant, UserAssignedMSI**)
-    - Set "ConnectionName" in the `appsettings.json`. The AAD ConnectionName from the OAuth Connection Settings on Azure Bot registration
+    - Set "ConnectionName" in the `appsettings.json`. The Microsoft Entra ConnectionName from the OAuth Connection Settings on Azure Bot registration
```
Comment (Pawank-MSFT) [inline samples/msgext-search-auth-config/csharp/README.md:88]: Microsoft Entra ConnectionName >> Microsoft Entra ID ConnectionName

**Review by Pawank-MSFT — COMMENTED** [review]
Please fix comments

File: samples/app-complete-auth/csharp/README.md [hunk 17]
```diff
@@ -103,7 +103,7 @@ Please find below demo manifest which is deployed on Microsoft Azure and you can
    - Select Configuration section.
    - Under configuration -> Add OAuth connection string.
    - Provide connection Name : for eg `ssoconnection`
-   - Select service provider ad `Azure Active Directory V2`
+   - Select service provider ad `Azure Active Directory v2`
```
[the remark this answers is not in the document]
Comment (Mohammed-MSFT) [reply samples/app-complete-auth/csharp/README.md:106]: fixed

File: samples/app-sso/csharp/README.md (hunk 1 above)
Comment (Mohammed-MSFT) [reply samples/app-sso/csharp/README.md:36]: fixed

File: samples/app-sso/nodejs/README.md (hunk 2 above)
Comment (Mohammed-MSFT) [reply samples/app-sso/nodejs/README.md:33]: fixed

File: samples/bot-conversation-sso-quickstart/csharp_dotnetcore/BotConversationSsoQuickstart/README.md (hunk 3 above)
Comment (Mohammed-MSFT) [reply samples/bot-conversation-sso-quickstart/csharp_dotnetcore/BotConversationSsoQuickstart/README.md:24]: fixed

File: samples/bot-conversation-sso-quickstart/js/README.md (hunk 4 above)
Comment (Mohammed-MSFT) [reply samples/bot-conversation-sso-quickstart/js/README.md:24]: fixed

File: samples/bot-daily-task-reminder/nodejs/Readme.md (hunk 5 above)
Comment (Mohammed-MSFT) [reply samples/bot-daily-task-reminder/nodejs/Readme.md:65]: fixed

File: samples/bot-daily-task-reminder/nodejs/Readme.md (hunk 6 above)
Comment (Mohammed-MSFT) [reply samples/bot-daily-task-reminder/nodejs/Readme.md:103]: fixed

File: samples/bot-formatting-cards/csharp/README.md (hunk 7 above)
Comment (Mohammed-MSFT) [reply samples/bot-formatting-cards/csharp/README.md:135]: Fixed

File: samples/bot-formatting-cards/nodejs/Readme.md (hunk 8 above)
Comment (Mohammed-MSFT) [reply samples/bot-formatting-cards/nodejs/Readme.md:123]: fixed In First PR

File: samples/bot-teams-authentication/csharp/README.md (hunk 9 above)
Comment (Mohammed-MSFT) [reply samples/bot-teams-authentication/csharp/README.md:23]: fixed in First PR

File: samples/bot-teams-authentication/nodejs/README.md (hunk 11 above)
Comment (Mohammed-MSFT) [reply samples/bot-teams-authentication/nodejs/README.md:23]: fixed in First PR

File: samples/graph-appcatalog-lifecycle/nodejs/README.md (hunk 13 above)
Comment (Mohammed-MSFT) [reply samples/graph-appcatalog-lifecycle/nodejs/README.md:173]: fixed

File: samples/graph-rsc/csharp/README.md (hunk 14 above)
Comment (Mohammed-MSFT) [reply samples/graph-rsc/csharp/README.md:46]: fixed

File: samples/graph-rsc/csharp/README.md (hunk 15 above)
Comment (Mohammed-MSFT) [reply samples/graph-rsc/csharp/README.md:75]: fixed

File: samples/app-sso/csharp/README.md (hunk 1 above)
Comment (Pawank-MSFT) [reply samples/app-sso/csharp/README.md:36]: Not fixed

Comment (Mohammed-MSFT) [reply samples/app-sso/csharp/README.md:36]: fixed

Comment (Mohammed-MSFT) [reply samples/app-sso/csharp/README.md:36]: fixed now

**Review by Prithvi-MSFT — APPROVED** [review]

**Review by Wajeed-msft — APPROVED** [review]

**Pull request merged**